### PR TITLE
Create CreatedbyHaley - 2nd

### PR DESCRIPTION
**Pull request by haleyhao-ca**
second file from Haley, please take a look

### CODE REVIEW

File: CreatedbyHaley - 2nd [hunk 1]
```diff
@@ -0,0 +1 @@
+a second file from Haley
```
Comment (satlilp) [inline CreatedbyHaley - 2nd:1]: can you please add something more to this file

Comment (satlilp) [inline CreatedbyHaley - 2nd:1]: please write more on this

Comment (haleyhao-ca) [reply CreatedbyHaley - 2nd:1]: done

Comment (haleyhao-ca) [reply CreatedbyHaley - 2nd:1]: done

**Pull request merged**